.changes/unreleased/Features-20230116-212648.yaml

```diff
@@ -0,0 +1,8 @@
+kind: Features
```

Comment:
Separate your submission in 2 commits:
- bumping the trino driver to `0.321.0`
- add`query_id` and `compiled_query` to adapter response

Reply:
Thanks for the guidance, i have update the above.